src/lib/swapper/swappers/ZrxSwapper/utils/helpers/helpers.ts

```diff
@@ -51,7 +50,7 @@ export const assertValidTradePair = ({
 }: {
   buyAsset: Asset
   sellAsset: Asset
-  adapter: ZrxSupportedChainAdapter
```

Comment:
Not sure about this one, that may be true currently but not all EVM chain adapters we support are guaranteed to be valid ZRX adapters

Reply:
well spotted ser, gnosis is an example of this. I've pushed a commit to prevent this swapper being used for unsupported chains.